hearts/hearts.py

```diff
@@ -239,10 +239,18 @@ def shot_the_moon(self):
                 return d
         return d
 
-    def final_scores(self):
+    def final_scores(self):     # FIXME:  test_final_scores_shot_the_moon() fails. 
```

Comment:
Is there a bug here?  test_final_scores_shot_the_moon() is failing, and it doesn't seem like `final_scores()` reaches the `else:` condition to modify `scores[player]`.  If there is an issue with modifying the dictionary values of `score`, or something simpler than that, I don't see it.